dev/devicelab/manifest.yaml

```diff
@@ -811,3 +811,18 @@ tasks:
   #   stage: devicelab_ios
   #   required_agent_capabilities: ["mac/ios", "ios/gl-render-image"]
   #   flaky: true
+
+  new_gallery_chrome_run_test:
+    description: >
+      Checks that the New Flutter Gallery runs successfully on Chrome.
+    stage: devicelab
+    required_agent_capabilities: ["linux/android"]
+    flaky: true
+
+  new_gallery_web_compile_test:
```

Comment:
gallery_v2_web_compile_test ?

Reply:
Done: changed to `flutter_gallery_v2_web_compile_test`.